src/Symfony/Component/DependencyInjection/Compiler/InlineCompilePass.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+namespace Symfony\Component\DependencyInjection\Compiler;
+
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+use Symfony\Component\DependencyInjection\Extension\CompilerExtensionInterface;
+
+/**
+ * A pass to execute inline compiler passes in extensions.
+ *
+ * @author Wouter J <wouter@wouterj.nl>
+ */
+class InlineCompilePass implements CompilerPassInterface
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function process(ContainerBuilder $container)
+    {
+        foreach ($container->getExtensions() as $extension) {
+            if (!$extension instanceof CompilerExtensionInterface) {
```

Comment:
The thing is that the `CompilerPassInterface` requires a method called `process()`, which I don't think is very describing in the Extension context.